javascript/javascript1/week1/housey pricey/housey-pricey.js

```diff
@@ -0,0 +1,21 @@
+
+
+name = "Peter";
```

Comment:
The `name` here and for Julia won't be doing anything in your code. If you would like to assign the labels "Peter" and "Julia" as variables you would need to use `const`

Or if it is simply to make it easier for you to read the sections then I recommend using comments

`// Peter's Section`

`// Julia's Section`